server/web/utils.py

```diff
@@ -0,0 +1,23 @@
+from flask_mail import Message, Mail
+from threading import Thread
+from flask import render_template, current_app
+
+mail = Mail()
+
+
+def send_async_email(app, msg):
+    with app.app_context():
+        mail.send(msg)
+
+
+def send_email(to, subject, template, **kwargs):
+    if isinstance(to, str):
+        to = [to]
+    app = current_app._get_current_object()
+    msg = Message(app.config['FLASKY_MAIL_SUBJECT_PREFIX'] + ' ' + subject,
```

Comment:
We can go ahead and remove the mail prefix. Doubtful we will use it nor do I think we should use it.

Reply:
We also don't have 'FLASKY_MAIL_SENDER', you want me change this as well?

Reply:
We also don't have 'FLASKY_MAIL_SENDER', you want me change it as well?